### DISCUSSION TITLE
How do I reply without marking the user's message?

### DISCUSSION THREAD

**Discussion by LucasSaud** [vasani-arpit/WBOT#347] [Q&A]
Hey goodnight! How do I reply without marking the user's message? I'll send you a print of how the response is going on whatsapp, I want to do it without including the user's text in the response. it's possible? Can you tell me what I need to change to be without the user message?

I thank you for your attention.

**Comment by vasani-arpit** [comment]
This feature is in pipeline, it is yet to be implemented.

**Comment by vasani-arpit** [comment]
This has been implemented. look for `quoteMessageInReply` in bot.json
